desktop/faqs/linuxfaqs.md

```diff
@@ -6,6 +6,165 @@ redirect_from:
+```
+
+### How do I switch between Docker Desktop and Docker Engine
+{: id="context" }
+
+The Docker CLI can be used to interact with multiple Docker Engines. For example,
+you can use the same Docker CLI to control a local Docker Engine and to control
+a remote Docker Engine instance running in the cloud. [Docker Contexts](../../engine/context/working-with-contexts.md)
+allow you to switch between Docker Engines instances.
+
+When installing Docker Desktop, a dedicated "desktop-linux" context is created to
+interact with Docker Desktop. On startup, Docker Desktop automatically sets its
+own context (`desktop-linux`) as the current context. This means that subsequent
+Docker CLI commands target Docker Desktop. On shutdown, Docker Desktop resets
+the current context to the `default` context.
+
+Use the `docker context ls` command to view what contexts are available on your
+machine. The current context is indicated with an asterisk (`*`);
+
+```console
+$ docker context ls
+NAME            DESCRIPTION                               DOCKER ENDPOINT                                  ...
+default *       Current DOCKER_HOST based configuration   unix:///var/run/docker.sock                      ...
+desktop-linux                                             unix:///home/<user>/.docker/desktop/docker.sock  ...        
+```
+
+If you have both Docker Desktop and Docker Engine installed on the same machine,
+you can run the `docker context use` command to switch between the Docker Desktop
+and Docker Engine contexts. For example, use the "default" context to interact
+with the Docker Engine;
+
+```console
+$ docker context use default
+default
+Current context is now "default"
+```
+
+And use the `desktop-linux` context to interact with Docker Desktop:
+
+```console
+$ docker context use desktop-linux
+desktop-linux
+Current context is now "desktop-linux"
+```
+
+Refer to the [Docker Context documentation](../../engine/context/working-with-contexts.md) for more details.
+
+## Why does Docker Desktop for Linux run a VM?
+
+Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
+
+1. **To ensure that Docker Desktop provides a consistent experience across platforms**.
+
+    During research, the most frequently cited reason for users wanting Docker Desktop for Linux (DD4L) was to ensure a consistent Docker Desktop
+    experience with feature parity across all major operating systems. Utilizing
+    a VM ensures that the Docker Desktop experience for Linux users will closely
+    match that of Windows and macOS.
+
+    This need to deliver a consistent experience across all major OSs will become increasingly important as we look towards adding exciting new features, such as Docker Extensions, to Docker Desktop that will benefit users across all tiers.  We’ll provide more details on these at [DockerCon22](https://www.docker.com/dockercon/){: target="_blank" rel="noopener" class="_"}. Watch this space.
```

Comment:
Let's reword this to make it future-proof! :-)

Reply:
Thank you for catching this @dvdksn!